test/conformance/enqueue/urEnqueueUSMMemcpy2D.cpp

```diff
@@ -1,2 +1,167 @@
 // Copyright (C) 2023 Intel Corporation
 // SPDX-License-Identifier: MIT
+
+#include "helpers.h"
+#include <uur/fixtures.h>
+
+struct urEnqueueUSMMemcpy2DTestWithParam
+    : uur::urQueueTestWithParam<uur::TestParameters2D> {
+    void SetUp() override {
+        UUR_RETURN_ON_FATAL_FAILURE(
+            uur::urQueueTestWithParam<uur::TestParameters2D>::SetUp());
+        const auto device_usm =
+            uur::GetDeviceInfo<bool>(device, UR_DEVICE_INFO_USM_DEVICE_SUPPORT);
+        ASSERT_TRUE(device_usm.has_value());
+        if (device_usm.value()) {
+            GTEST_SKIP() << "device USM not supported.";
+        }
```

Comment:
Shouldn't this be `if(!device_usm.value())` ?

Reply:
Done.